ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_loss.py

```diff
@@ -391,3 +391,55 @@ def test_paddle_margin_ranking_loss(
         margin=margin,
         reduction=reduction,
     )
+
+
+@handle_frontend_test(
+    fn_tree="paddle.nn.functional.triplet_margin_loss",
+    dtype_and_inputs=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=3,
+        allow_inf=False,
+        shared_dtype=True,
+        min_value=0.0,
+        max_value=1.0,
+        min_num_dims=1,
+        max_num_dims=2,
+        min_dim_size=1,
+    ),
+    margin=st.floats(min_value=1e-6, max_value=1e6),
+    p=st.integers(min_value=0, max_value=2),
+    swap=st.booleans(),
+    reduction=st.sampled_from(["none", "mean", "sum"]),
+    test_with_out=st.just(False),
+)
+def test_paddle_triplet_margin_loss(
+    dtype_and_inputs,
+    margin,
+    p,
+    swap,
+    reduction,
+    test_flags,
+    fn_tree,
+    backend_fw,
+    frontend,
+    on_device,
+):
+    input_dtype, x = dtype_and_inputs
+    anchor_dtype, input = input_dtype[0], x[0]
+    positive_dtype, positive = input_dtype[1], x[1]
+    negative_dtype, negative = input_dtype[2], x[2]
+    helpers.test_frontend_function(
+        input_dtypes=[anchor_dtype, positive_dtype, negative_dtype],
```

Comment:
It's fine to pass the `input_dtype` variable here not necessarily to unpack all those variables above